db/dcrpg/pgblockchain.go

```diff
@@ -560,6 +523,9 @@ func NewChainDBWithCancel(ctx context.Context, dbi *DBInfo, params *chaincfg.Par
 		hash:   bestHash,
 	}
 
+	addrCache := cache.NewAddressCache(500)
+	addrCache.DevAddress = devSubsidyAddress
```

Comment:
project fund address is set to prevent purging its data from cache when cache reaches capacity.